### PR TITLE
Changes in image for stateless test flaky check

### PR DESCRIPTION
**Pull request by alesapin**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Add flaky check for stateless tests

### CODE REVIEW

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -16,9 +16,17 @@ service clickhouse-server start && sleep 5
...
+function run_tests()
+{
+    for i in $(seq 1 $NUM_TRIES); do
+        clickhouse-test --testname --shard --zookeeper --hung-check --print-time "$SKIP_LIST_OPT" "${ADDITIONAL_OPTIONS[@]}" "$SKIP_TESTS_OPTION" 2>&1 | ts '%Y-%m-%d %H:%M:%S' | tee -a test_output/test_result.txt
+    done
+}
+
+export -f run_tests
+
+timeout $MAX_RUN_TIME bash -c run_tests ||:
```
Comment (alexey-milovidov) [inline docker/test/stateless/run.sh:32]: But 0 does not mean unlimited here.

**Comment by alexey-milovidov** [comment]
It's the best place where we can also add Thread Fuzzer...

File: docker/test/stateless/run.sh (hunk 1 above)
Comment (alesapin) [reply docker/test/stateless/run.sh:32]:
```
$ timeout 0 sleep 10
$ echo $?
0
```
?

Comment (alexey-milovidov) [reply docker/test/stateless/run.sh:32]: Ok.

man timeout
> A duration of 0 disables the associated timeout.

**Comment by alesapin** [comment]
What's wrong with tests?

**Comment by alexey-milovidov** [comment]
> test_dictionaries_dependency_xml

Unknown issue.

**Pull request merged**